tests/playwright-test/reporter-blob.spec.ts

```diff
@@ -947,6 +947,59 @@ result.stderr: stderr text
 `);
 });
 
+test('encode inline attachments', async ({ runInlineTest, mergeReports }) => {
+  const reportDir = test.info().outputPath('blob-report');
+  const files = {
+    'echo-reporter.js': `
+      import fs from 'fs';
+
+      class EchoReporter {
+        onTestEnd(test, result) {
+          const attachmentBodies = result.attachments.map(a => a.body?.toString('base64'));
+          result.attachments.forEach(a => console.log(a.body, 'isBuffer', Buffer.isBuffer(a.body)));
```

Comment:
Where do we check this output? Should we remove it?

Reply:
Removed.